src/e2e-tests/page-objects/smartthings/officeTab.page.js

```diff
@@ -0,0 +1,69 @@
+// officeTab.page.js
+import Page from '../page';
+import MenuComponent from '../menu.component';
+
+class OfficeTabPage extends Page {
```

Comment:
shall not extend page because it is not a page, it is more a component, set of options

Reply:
Updated.